### PR TITLE
[ko] Update outdated files in `dev-1.23-ko.2` (M1-M8)

### PR DESCRIPTION
**Pull request by jihoon-seo**
한 PR에 너무 많은 변경사항을 포함시키면 리뷰하기 어려울 수 있어서, 적절히 분할하여 올립니다. 😊

- Related issue: #31647 (**dev-1.23-ko.2**)

> ### 8 files to be modified
> * [ ]  M1.  `content/en/_index.html`  | 1(+XS) 1(-)
> * [ ]  M2.  `content/en/docs/concepts/architecture/cloud-controller.md`  | 1(+XS) 1(-)
> * [ ]  M3.  `content/en/docs/concepts/architecture/cri.md`  | 1(+XS) 1(-)
> * [ ]  M4.  `content/en/docs/concepts/architecture/nodes.md`  | 2(+XS) 2(-)
> * [ ]  M5.  `content/en/docs/concepts/cluster-administration/logging.md`  | 1(+XS) 1(-)
> * [ ]  M6.  `content/en/docs/concepts/cluster-administration/networking.md`  | 13(+S) 47(-)
> * [ ]  M7.  `content/en/docs/concepts/cluster-administration/system-logs.md`  | 2(+XS) 1(-)
>   * `2(+XS) 1(-)`로 나와 있는데, 확인해 보니 더 outdated 되어 있어 update 했습니다.
> * [ ]  M8.  `content/en/docs/concepts/configuration/configmap.md`  | 4(+XS) 0(-)

/language ko

### CODE REVIEW

**Comment by seokho-son** [comment]
@ClaudiaJKang PR 리뷰 가능하실까요? ^^

**Review by ClaudiaJKang — COMMENTED** [review]
리뷰 완료했습니다!

/lgtm

File: content/ko/docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -30,46 +30,7 @@ weight: 50
 찾는 방법 등을 알아야 한다. 쿠버네티스는 이런 것들을 다루는 대신
 다른 접근법을 취한다.
 
-## 쿠버네티스 네트워크 모델
```
Comment (ianychoi) [inline content/ko/docs/concepts/cluster-administration/networking.md:33]: Note: L33-72 was migrated to another document - will address via another issue/PR

**Review by ianychoi — COMMENTED** [review]
/lgtm

**Pull request merged**